index.js

```diff
@@ -2,7 +2,19 @@ module.exports = {
   rules: {
     'jsx-needs-i18n': require('./lib/rules/jsx-needs-i18n')
   },
-  rulesConfig: {
```

Comment:
rulesConfg support removed in 2.0.0
http://eslint.org/docs/user-guide/migrating-to-2.0.0